src/main/java/com/google/sps/servlets/MatchServlet.java

```diff
@@ -39,11 +39,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public synchronized void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException {
 
-    String[] matchesToSave = request.getParameterValues("new-matches");
+    if(request.getParameterValues("new-matches") != null) {
```

Comment:
maybe include a test that would fail without this null check?

Reply:
Good idea, I added one!